packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -100,6 +102,60 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody> {
       ],
     );
   }
+
+  Widget debuggerPanes() {
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        final paneWidth = constraints.maxWidth;
+        return FlexSplitColumn(
+          totalHeight: constraints.maxHeight,
```

Comment:
seems a bit odd we have to pass in the totalHeight to the FlexSplitColumn. This seems like the natural default height for a FlexSpltColumn same as it would be for a regular column

Reply:
we need `totalHeight` to perform calculations in a static method `FlexSplitColumn.adjustInitialFractions`, not for building the widget.